### DISCUSSION TITLE
TeslaMate / Drives   page,all the drive log are mess

### DISCUSSION THREAD

**Discussion by zhihz** [teslamate-org/teslamate#1102] [Q&A]
![企业微信截图_20201130080438](https://user-images.githubusercontent.com/3733774/100557237-c0963800-32e2-11eb-863f-5e31197e003b.png)

**Comment by zhihz** [comment]
HELP ME? can i fix it?

**Comment by breez7** [comment]
help me too

**Comment by RobReus** [comment]
Same issue here
![img](https://i.imgur.com/0E0vXgk.png)

Running on version 1.20.1
